ignite/metrics/metric.py

```diff
@@ -233,6 +233,58 @@ class Metric(Serializable, metaclass=ABCMeta):
         device: specifies which device updates are accumulated on. Setting the
             metric's device to be the same as your ``update`` arguments ensures the ``update`` method is
             non-blocking. By default, CPU.
+        skip_unrolling: specifies whether output should be unrolled before being fed to update method. Should be
+            true for multi-output model, for example, if ``y_pred`` contains multi-ouput as ``(y_pred_a, y_pred_b)``
+
+            Example usage:
+                The following example shows a custom loss metric that expects input from a multi-output model.
+                .. code-block:: python
+                    import torch
+                    import torch.nn as nn
+                    import torch.nn.functional as F
```

Comment:
I think we should reformat here to fix docs tests:
```suggestion
            Examples:
            
                The following example shows a custom loss metric that expects input from a multi-output model.
            
                .. code-block:: python

                    import torch
                    import torch.nn as nn
                    import torch.nn.functional as F
```

See for example here: https://github.com/pytorch/ignite/blob/5a66d9e09fad42edb1fdda88a426a8658876fedd/ignite/engine/engine.py#L38-L46